### PR TITLE
added new files

### PR DESCRIPTION
**Pull request by Abysemilla97**
Please review the files.

### CODE REVIEW

File: char_print.c [hunk 1]
```diff
@@ -0,0 +1,13 @@
+#include main.h
+#include <stdio.h>
+
+/**
+ * my_charprint - a function that print characters.
+ * @count: counting the character. 
+ * Return: nothing 0.
```
Comment (Tednic) [inline char_print.c]: This function returns the size of nbytes of character.

Comment (Tednic) [reply char_print.c]: Plz correct

File: print_percent.c [hunk 2]
```diff
@@ -0,0 +1,13 @@
+#include "main.h"
+#include <stdio.h>
+
+/**
+ * my_printpercentage - a function that prints percentage.
+ * @argc: count characters. 
+ * Return: nothing 0.
```
Comment (Tednic) [inline print_percent.c]: Same as the above..

File: string_print.c [hunk 3]
```diff
@@ -0,0 +1,17 @@
+#include "main.h"
+#include <stdio.h>
+
+/**
+ */
```
Comment (Tednic) [inline string_print.c:8]: Add description

**Review by Abysemilla97 — COMMENTED** [review]
I have resolved the issues.

**Comment by Tednic** [comment]
Ok merge!!
You too much!!!

**Pull request merged**